client-code-gen/gen/app-access-control-api/api/famuser-terms-and-conditions-api.ts

```diff
@@ -92,10 +84,6 @@ export const FAMUserTermsAndConditionsApiAxiosParamCreator = function (configura
             // oauth required
             await setOAuthToObject(localVarHeaderParameter, "6jfveou69mgford233or30hmta", [], configuration)
```

Comment:
Should this "6jfv..." value be a constant? What is this for?

Reply:
Hi Basil, this file is one of the generated frontend api-client code, it is internal to the generated code that we don't control it and don't directly use it frontend or backend. 
The value "**6jfv**..." by itself looks like is one of the environment variable ("**COGNITO_CLIENT_ID**"), the Cognito client for FAM. How it gets into generated code is unknown to me, but it seems the OpenAPI/Swagger is smart enough to grab that when server is being served up, and it seems Swagger uses this value to describe the "**securitySchemes**".
The value previously already exists and did not appear to cause any issue. This is the "openapi.json" that contains the value:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/1b78c5f4-7b1e-4c23-9b37-6ba7acc53d4b)

Reply:
Ah, okay, I missed that this was a generated file when I was reviewing, that makes more sense.